swiss-ng-api/berlin-group-documentation/current.yaml

```diff
@@ -7909,13 +8082,28 @@ components:
         - "ERROR"
         - "WARNING"
 
+
     MessageCode2XX: 
       description: Message codes for HTTP Error codes 2XX.
       type: string
       enum: 
         - "WARNING"
 
 
+    MessageCode202InitiationStatus: 
+      description: Message codes for HTTP codes 202 to a Payment Initiation Status Request.
+      type: string
+      enum: 
+        - "FUNDS_NOT_AVAILABLE"
+
+
+    MessageCode201PaymentInitiation:
+      description: Message codes for HTTP Codes 201 to a Payment Initiation Request.
+      type: string
+      enum: 
+        - "WARNING"
+        - "BENEFICIARY_WHITELISTING_REQUIRED"
```

Comment:
Relevant Change: MessageCode201PaymentInitiation is new.
Adoption approach: Follow the change